### ISSUE TITLE
Refunds and Purchase ID

### ISSUE THREAD

**Issue by oorlov** [onepf/OpenAEP#9]
If application is sold by another store and user asked for refund there is no way to share this data to source store

**Comment by jesper-sjovall** [comment]
Yes, this is a problem, but how big? How common is this a user want to refund a app?
And for the second if the system as like Google Play have 15 minuter refund time this is not very probably other stores
has yet got this informations if not continuously try to get new data.

**Comment by akarimova** [comment]
Notes:
-  What if the application - that sold an item - was deleted by the user? 
-  Refund stats for the known stores
